app/views/layouts/_item_form.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= form_with model: @item, local: true do |f| %>
```

Comment:
Not completely sure how this looks when rendered, but I'm wondering if there is a way we can have the existing information about the item be filled into the form by default instead of having the user fill it all out again (if it already does this I'm sorry lol). If not, we can implement in another pr.

Reply:
Yes, I think user#edit has this implemented right now but I'll check

Reply:
Yes it does, take a look for reference if you want! @mrubakh

Reply:
Yep I just want to confirm the the info is already there, feel free to test it out yourself :)